Sandbox/Sandbox/DevicesFidoCell.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  DevicesFidoCell.swift
+//  Sandbox
+//
+//  Created by admin on 04/03/2021.
+//  Copyright © 2021 Reachfive. All rights reserved.
+//
+
```

Comment:
Remove this. I think there might be a way to remove this from the template (cf `/Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/File Templates`.)

Reply:
Done

Reply:
FYI there's probably a way to edit those templates, see [this](https://stackoverflow.com/a/10097202/1422249) stackoverflow answer for an example.